src/Modules/ContentBlocks/Domain/ContentBlock/ContentBlock.php

```diff
@@ -0,0 +1,227 @@
+<?php
+
+namespace ForkCMS\Modules\ContentBlocks\Domain\ContentBlock;
+
+use Doctrine\ORM\Mapping as ORM;
+use ForkCMS\Modules\Backend\Domain\Action\ModuleAction;
+use ForkCMS\Modules\Internationalisation\Domain\Locale\Locale;
+use DateTime;
+use Pageon\DoctrineDataGridBundle\Attribute\DataGrid;
+use Pageon\DoctrineDataGridBundle\Attribute\DataGridActionColumn;
+use Pageon\DoctrineDataGridBundle\Attribute\DataGridMethodColumn;
+use Pageon\DoctrineDataGridBundle\Attribute\DataGridPropertyColumn;
+
+#[ORM\Entity(repositoryClass: ContentBlockRepository::class)]
+#[ORM\Table(name: 'contentblocks__contentblock')]
+#[DataGrid('ContentBlock')]
+#[ORM\HasLifecycleCallbacks]
+#[DataGridActionColumn(
+    route: 'backend_action',
+    routeAttributes: [
+        'module' => 'content_blocks',
+        'action' => 'content_block_edit',
+    ],
+    routeAttributesCallback: [self::class, 'dataGridEditLinkCallback'],
+    label: 'lbl.Edit',
+    class: 'btn btn-primary btn-sm',
+    iconClass: 'fa fa-edit',
+    requiredRole: ModuleAction::ROLE_PREFIX . 'CONTENT_BLOCKS__CONTENT_BLOCK_EDIT',
+    columnAttributes: ['class' => 'fork-data-grid-action'],
+)]
+final class ContentBlock
+{
+    const DEFAULT_TEMPLATE = 'Default.html.twig';
+
+    #[ORM\Id]
+    #[ORM\GeneratedValue]
+    #[ORM\Column(name: 'revision_id', type: 'integer')]
+    private int $revisionId;
+
+    #[ORM\Column(type: 'integer')]
+    private int $id;
+
+    #[ORM\Column(name: 'user_id', type: 'integer')]
+    private int $userId;
+
+    #[ORM\Column(name: 'extra_id', type: 'integer')]
+    private int $extraId;
+
+    #[ORM\Column(type: 'string', length: 255, options: ['default' => 'Default.html.twig'])]
+    private string $template;
+
+    #[ORM\Column(name: 'language', type: 'string', length: 5, enumType: Locale::class)]
+    private Locale $locale;
+
+    #[ORM\Column(type: 'string', length: 255)]
+    #[DataGridPropertyColumn(
+        sortable: true,
+        label: 'lbl.Title',
+        route: 'backend_action',
+        routeAttributes: [
+            'module' => 'content_blocks',
+            'action' => 'content_block_edit',
+        ],
+        routeAttributesCallback: [self::class, 'dataGridEditLinkCallback'],
+        routeRole: ModuleAction::ROLE_PREFIX . 'CONTENT_BLOCKS__CONTENT_BLOCK_EDIT',
+        columnAttributes: ['class' => 'title'],
+    )]
+    private string $title;
+
+    #[ORM\Column(type: 'text', nullable: true)]
+    private ?string $text;
```

Comment:
Making this nullable doesn't make sense to me. should be required

Reply:
https://github.com/forkcms/forkcms/pull/3555/commits/c24fac573aad3d9e429b9a352c24087b2dd1f862